chart/kubeapps/templates/apprepository-jobs-cleanup-rbac.yaml

```diff
@@ -0,0 +1,47 @@
+{{- if .Values.rbac.create -}}
```

Comment:
Ditto previous comment about adding some context on why this this job is needed in a comment.

Reply:
added in all the `cleanup` related files.